src/App.js

```diff
@@ -1,43 +1,25 @@
 //TODO: STEP 1 - Import the useState hook.
-import React from "react";
+import React, {useState} from "react";
 import "./App.css";
-import BottomRow from "./BottomRow";
+import Scoreboard from "./Scoreboard.js";
+import Buttons from "./Buttons.js";
+// import Scoreboard, {homeScore, setHomeScore, awayScore, setAwayScore} from "./Scoreboard.js";
```

Comment:
We talked about this, but we would use `props` to pass data around rather than using `import`s